.circleci/config.yml

```diff
@@ -225,7 +264,30 @@ jobs:
           root: /app
           paths:
             - core-<<parameters.language>>
-
+  dev_test_integration:
+    description: Run integration tests against deployed dev service.
+    parameters:
+      <<: *param__working_directory
+      <<: *param__language
+    working_directory: ~/project/<<parameters.working_directory>>
+    executor: python/python
+    steps:
+      - setup-project
+      - run:
+          name: Run integration tests
+          command: CORE_ENDPOINT_URL=https://${CIRCLE_PROJECT_REPONAME}-pr-${CIRCLE_PR_NUMBER}-core-<<parameters.language>>.dev.dialoguecorp.com poetry run python -m rasa_integration_testing ../integration-tests-<<parameters.language>>
+  master_test_integration:
+    description: Run integration tests against deployed service on master.
+    parameters:
+      <<: *param__working_directory
+      <<: *param__language
+    working_directory: ~/project/<<parameters.working_directory>>
+    executor: python/python
+    steps:
+      - setup-project
+      - run:
+          name: Run integration tests
+          command: CORE_ENDPOINT_URL=https://${CIRCLE_PROJECT_REPONAME}-core-<<parameters.language>>.dialoguecorp.com poetry run python -m rasa_integration_testing ../integration-tests-<<parameters.language>>
```

Comment:
from what I see, you can have a single job `test_integration` which takes a `core_endpoint_url` in parameter instead of duplicating the whole thing twice. (dont forget to change job description :) )

Reply:
Done